source/language/openpulse.rst

```diff
@@ -66,30 +73,16 @@ device) and receive ports (reading output from a quantum device).
 A port is only used to specify the physical resource on which to play a pulse or from which
 to capture data. This specification should be done by providing a mapping between a qubit list +
 name and the configured hardware port. The hardware can then be accessed as
-OpenPulse ``port``'s via the "getport" function which allows identifying a port by a variadic combination
-of string name and qubits.
+OpenPulse ``port``'s via ``extern`` identifier to specify an external linkage
```

Comment:
Do we also need this to be supported for frames so that we can link into existing calibrations?

Reply:
Yes I think so. I'll fix that.